data/tabular/ames_mutagenicity/transform.py

```diff
@@ -138,7 +138,7 @@ def get_and_transform_data():
 Constraint: Answer the question in a {#full|complete!} sentence.
 Result: This molecule is {mutagenic#not &NULL}{mutagenic__names__adjective}.""",
             """Task: Please {#give me|create|generate!} a {#molecule |!}{SMILES__description} based on the {#text |!}description{# below|!}.
-Description: A molecule that is {mutagenic__names__adjective}.
+Description: A molecule that is {mutagenic#not &NULL}{mutagenic__names__adjective}.
```

Comment:
do we have an easy way to automatically verify it (in the future?)

Reply:
just because you now changed this in a few files (and it is very hard for me to verify and also spot)

Reply:
Not really, you have to know what the raw labels are meaning.
But in those fixes the label needs to change according to the input, so this was missing.